content/policies/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: npm Policies
+---
+
+These are updated from time to time.  Their sources are stored in a git
+repository at [https://github.com/npm/policies](https://github.com/npm/policies).
```

Comment:
This might be an issue with how the docs site is setup but the current [policies page](https://www.npmjs.com/policies/) only lists some of the pages and uses specific titles for them that are inconsistent with this implementation.

Below are the links + titles as visible on the current policies page

> * Terms of Use
> * Code of Conduct
> * Package Name Disputes
> * npm License
> * Privacy Policy
> * Unpublish Policy
> * Receiving Abuse Reports
> * Copyright and DMCA Policy
> * Trademark Policy
> * Security
> * Replication and web crawler policy
> * Our official list of domains